### PR TITLE
docs: adding CONTRIBUTING.md #59

### PR DESCRIPTION
**Pull request by SPlanzer**
Fixes: #  
#59

### Change Description:
adds  CONTRIBUTING.md which includes details about how others can contribute to the project. 

### Notes for Testing:

...

#### Source Code Documentation Tasks:
- [x] README updated (where applicable)
- [x] CHANGELOG updated

#### User Documentation Tasks:
- [x] Confluence user guide updated (where applicable)

#### Testing Tasks:
- [x] Added tests that fail without this change
- [x] All tests are passing in development environment
- [x] Reviewers assigned

#### Pull Request Management: 
- [x] Linked to sub-task
- [x] Linked to the epic

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# Contributing to Basemaps
```
Comment (palmerj) [inline CONTRIBUTING.md]: replace basemaps with NZGB app

Comment (SPlanzer) [reply CONTRIBUTING.md]: thanks @palmerj 
Done

**Comment by SPlanzer** [comment]
@strk as discussed, here is one of the PR I required reviewed / approved so that I can merge to master

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+Linting is handled by `pylint`.
+
+[Pylint](https://www.pylint.org/) checks Python files in order to detect syntax errors and potential bugs (unreachable code / unused variables), provide refactoring help,
+
+The configuration is stored in .pylintrc.
+
+## Commit message
+
+This repository uses Conventional Commits
```
Comment (strk) [inline CONTRIBUTING.md]: as I've never heard of `Conventional Commits`, it would help if a link was given to a formal document about it..

**Pull request merged**